membership-common/src/main/scala/com/gu/memsub/subsv2/Plan.scala

```diff
@@ -122,178 +108,121 @@ case object Years extends UpToPeriodsType {
 }
 
 object UpToPeriodsType {
-  val values = Seq(BillingPeriods, Days, Weeks, Months, Years)
+  val values: Seq[UpToPeriodsType] = Seq(BillingPeriods, Days, Weeks, Months, Years)
   implicit val reads: Reads[UpToPeriodsType] = ZuoraEnum.getReads(values, "invalid up to periods type value")
 }
 
 /** Low level model of a Zuora rate plan charge
   */
-case class ZuoraCharge(
+case class RatePlanCharge(
     id: SubscriptionRatePlanChargeId,
     productRatePlanChargeId: ProductRatePlanChargeId,
     pricing: PricingSummary,
-    billingPeriod: Option[ZBillingPeriod],
+    zBillingPeriod: Option[ZBillingPeriod],
```

Comment:
this field is renamed and the old low->high level deserialiser is moved to the billingPeriod method below
used to be 
implicit def anyBpReads[B <: BillingPeriod: ClassTag]: ChargeReads[B]
membership-common/src/main/scala/com/gu/memsub/subsv2/reads/ChargeListReads.scala